src/vl-map-search.js

```diff
@@ -181,8 +227,20 @@ export class VlMapSearch extends vlElement(HTMLElement) {
 
   _changeTranslations() {
     this.placeholder = 'Lokaliseer adres';
-    this.searchPlaceholder = 'Gemeente, straat en huisnummer';
+    this.searchPlaceholder = 'Gemeente, straat en huisnummer of Lambert coördinaat';
     this.searchEmptyText = 'Geen adres gevonden';
     this.searchNoResultsText = 'Geen adres gevonden';
   }
+
+  /**
+   * Find reverse geocoded addresses based upon a Lambert-72 coordinate.
+   * A maximum of 5 addresses will be returned.
+   *
+   * @param {LambertCoordinaat} lambertCoordinaat
+   * @return {string}
+   * @see {@link https://loc.geopunt.be/Help/Api/GET-v4-Location_q_latlon_xy_type_c|Request Information}
+   */
+  _findLocationsByLambertCoordinaatUrl(lambertCoordinaat) {
+    return this.url + '/Location?xy=' + lambertCoordinaat.x + ',' + lambertCoordinaat.y + '&c=5';
```

Comment:
Ik zie dat this.url een constante is, dus op zich zijn we relatief zeker dat het wel of niet eindigt met een slash, maar doorgaans zie ik liever dat URLs op een andere manier opgemaakt worden dan met string concatenatie om foutjes te voorkomen.

Reply:
Je zou een literal kunnen gebruiken om de leesbaarheid te verhogen. Ik zou ook een getter introduceren net zoals `searchUrl` en `locationUrl.